felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1009,7 +1013,9 @@ func (m *bpfEndpointManager) calculateTCAttachPoint(policyDirection PolDirection
 	ap.PSNATEnd = m.psnatPorts.MaxPort
 	ap.IPv6Enabled = m.ipv6Enabled
 	ap.MapSizes = m.bpfMapContext.MapSizes
-
+	if m.Features != nil {
+		ap.Features = *m.Features
```

Comment:
What if `Features == nil` ? Then the `ap.Features` are set correctly to default values?

Reply:
the default value was `false` which meant `tnl` program was used. Now the default value is set based on kernel version.